aws/modules_common.tf

```diff
@@ -15,12 +15,11 @@ module "codebuild_cd-filecoin-fluent-bit" {
   ]
 }
 
-module "codebuild_ci_cid-checker" {
+module "codebuild_cd_cid-checker_watcher_mainnet" {
```

Comment:
I know it is more about naming. but don't need to put **_mainnet** in the module name without any strong reason.
It can be a reason for misunderstanding logic and where it has to be deployed. 
Moreover, we have the count attribute that explicitly shows where it will be deployed